src/fetch_family_allowance.py

```diff
@@ -0,0 +1,91 @@
+import datetime
+import warnings
+import requests
+import zipfile
+from urllib.request import urlretrieve
+import os
+import pandas
+import fileinput
+from sqlalchemy import create_engine
+import sys
+import shutil
+
+STARTING_YEAR = 2011
+
+NOW = datetime.datetime.now()
+
+SERVER = 'http://arquivos.portaldatransparencia.gov.br/'
+PATH = 'downloads.asp?a={}&m={:02d}&consulta=BolsaFamiliaFolhaPagamento'
+outputfile = 'data_family_allowance.csv'
```

Comment:
All constants are in caps, can we have that one in caps too?

Also we don't seem to use `outputfile` alone: it's always `BASE_DATA_DIR + outputfile` — what about concatenating `BASE_DATA_DIR` to the output file constant?